### PR TITLE
Specify limits on pending sources per source origin

### PR DESCRIPTION
**Pull request by apasel422**
https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#storage-limits

#386

***
<a href="https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/412.html" title="Last updated on May 10, 2022, 7:08 PM UTC (6424877)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/412/f857acd...apasel422:6424877.html" title="Last updated on May 10, 2022, 7:08 PM UTC (6424877)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -636,6 +636,12 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. [=list/Remove=] all entries in |cache| where the entry's [=attribution source/expiry time=] is less than the current time.
 1. If the [=list/size=] of |cache| is greater than or equal to the user agent's
      [=max source cache size=], return.
+1. Let |pendingSourcesForSourceOrigin| be the [=set=] of all
+    [=attribution sources=] |pendingSource| of |cache| where |pendingSource|'s
```
Comment (csharrison) [inline index.bs:640]: s/of |cache|/in |cache|/

File: index.bs [hunk 2]
```diff
@@ -636,6 +636,12 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
...
+    [=attribution source/source origin=] and |source|'s
+    [=attribution source/source origin=] are equal.
```
Comment (csharrison) [inline index.bs:642]: Should we say "same origin" here to be precise?
https://html.spec.whatwg.org/multipage/origin.html#same-origin

We already disallow opaque origins so it should be equivalent, and I don't think there's a standard "origin equality"

**Review by csharrison — APPROVED** [review]
LGTM with two suggestions

File: index.bs (hunk 1 above)
Comment (apasel422) [reply index.bs:640]: Per https://infra.spec.whatwg.org/#list-iterate, `of` is the standard term.

Comment (csharrison) [reply index.bs:680]: Thank you for the reference! My mistake.

**Pull request merged**